common/header/components/navigation/navigation.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC, memo, useId } from 'react';
+
+import NavigationLink, { INavigationLink } from './components/navigationLink';
+import { NavigationStyle } from './styles';
+import { INavigation } from './types';
+
+const Navigation: FC<INavigation> = memo((props: INavigation) => {
+  const { pages } = props;
+
+  const id = useId();
+
+  if (!pages) {
+    return <></>;
+  }
+
+  return (
+    <NavigationStyle>
+      {pages.map((pageProps: INavigationLink, index: number) => (
+        <NavigationLink key={`${id}-${index}`} {...pageProps} />
+      ))}
+    </NavigationStyle>
+  );
+});
+
+// TODO: ???
```

Comment:
for myself. without 'Navigation.displayName = 'Navigation';' not working. I want to understand why.

Reply:
Can you add context in the comment? In case, someone else will be editing the same component.

Btw wasn't able to reproduce it locally, when removed next line

`yarn build`
<img width="1081" alt="image" src="https://user-images.githubusercontent.com/103929444/211805111-6c59b168-1bb5-4f2f-b2ea-3944972e2c24.png">

`yarn dev`
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/103929444/211805162-291ea5cc-e40e-4aeb-b926-89db09fc3cc4.png">